2.8/debian-10/rootfs/opt/bitnami/scripts/kafka/setup.sh

```diff
@@ -21,8 +21,8 @@ if [[ -z "${KAFKA_CFG_BROKER_ID:-}" ]]; then
     if [[ -n "${BROKER_ID_COMMAND:-}" ]]; then
         KAFKA_CFG_BROKER_ID="$(eval "${BROKER_ID_COMMAND:-}")"
         export KAFKA_CFG_BROKER_ID
-    else
-        # By default auto allocate broker ID
+    elif [[ -z "${KAFKA_ENABLE_KRAFT:-}" ]]; then
```

Comment:
Once `KAFKA_ENABLE_KRAFT` is added to [`kafka-env.sh`](https://github.com/bitnami/bitnami-docker-kafka/blob/master/2.8/debian-10/rootfs/opt/bitnami/scripts/kafka-env.sh), we could simply use it like this:

```suggestion
    elif [[ -z "$KAFKA_ENABLE_KRAFT" ]]; then
```